projects/go-lib/src/lib/components/go-layout/go-layout.component.html

```diff
@@ -6,13 +6,13 @@
     </nav>
     <div class="go-layout__view-container">
 
-      <go-header-bar #headerBar [@routerAnimation]="routeAnimation(mainRouter)"></go-header-bar>
```

Comment:
**This** is actually the culprit of why this issue was happening. Angular was performing route animations on the header bar, but the positioning was off with the header bar. Rather than investigating a clean solution to absolutely positioning the elements inside of the header bar, I opted to just remove the animation for now.